src/WCSimPMTObject.cc

```diff
@@ -1699,7 +1699,7 @@ G4String HPD20inchHQE::GetPMTName() {G4String PMTName = "HPD20inchHQE"; return P
 G4double HPD20inchHQE::GetExposeHeight() {return .192*m;}
 G4double HPD20inchHQE::GetRadius() {return .254*m;}
 G4double HPD20inchHQE::GetPMTGlassThickness() {return 0.3*cm;}
-double HPD20inchHQE::HitTimeSmearing(double Q, double TTSFF=1.0) {
+double HPD20inchHQE::HitTimeSmearing(double Q, double=1.0) { // TTSFF not used in this implementation
```

Comment:
This is an unclear comment now
```suggestion
double HPD20inchHQE::HitTimeSmearing(double Q, double /*TTSFF*/=1.0) { // TTSFF not used in this implementation
```
or alternatively
```
// TTSFF (argument 2) not used in this implementation
```
Same comment for the other instances of this

Reply:
@tdealtry I agree.  I committed a more verbose version:
```
// Implementation of pure-virtual base-class method
//   double WCSimPMTObject::HitTimeSmearing(double, double)
// The second argument is usually a multiplcative factor named TTSFF, but that
// quantity is not used in this implementation.  The unused second argument
// is anonymous to prevent compiler warnings about unused arguments.
```